include/nuttx/fs/fs.h

```diff
@@ -461,7 +462,7 @@ struct file
 {
   int               f_oflags;   /* Open mode flags */
 #ifdef CONFIG_FS_REFCOUNT
-  int               f_refs;     /* Reference count */
+  atomic_t          f_refs;     /* Reference count */
```

Comment:
let's create a new patch to remove fl_crefs at line 498

Reply:
ok